.env.example

```diff
@@ -1,3 +1,7 @@
+TWILIO_ACCOUNT_SID=
+TWILIO_AUTH_TOKEN=
+TWILIO_PHONE_NUMBER=
+
 APP_ENV=local
 APP_KEY=
 APP_DEBUG=true
```

Comment:
Please remove everything that is not needed in this file (cache, Redis, Mail) and fix default DB parameters to use sqlite. Probably remove DB username and password